cmd/http/main.go

```diff
@@ -227,9 +215,23 @@ func (handler *KeyValueHandler) createKV(w http.ResponseWriter, r *http.Request)
 
 	rev := revision.GetGlobalIncreasingRevision()
 	newRev := index.NewRevision(int64(rev), 0, nil)
-	node := handler.ch.LocateKey(handler.getPrimaryRevBytesWithBucket(newRev))
-	newRev.SetNodes(strings.Split(node.String(), ","))
+	lns, rns, err := handler.hm.GetLocalAndRemoteNodes(handler.getPrimaryRevBytesWithBucket(newRev))
```

Comment:
suggest to name as local (or localNodes)

Reply:
Updated